keyboard_rewrite/src/lib/y2_keyboard/keyboard_layout.rb

```diff
@@ -0,0 +1,26 @@
+require "cheetah"
+
+module Y2Keyboard
+  class KeyboardLayout
+    attr_reader :code
+    attr_reader :description
+
+    LAYOUT_CODE_DESCRIPTIONS = {
+      "es" => "Spanish",
+      "fr" => "French",
+      "us" => "English (US)"
+    }
+
+    def initialize(code, description)
+      @code = code
+      @description = description
+    end
+
+    def self.load
```

Comment:
I would prefer a name like `all` so the API will look like like:

```ruby
Y2Keyboard::KeyboardLayout.all # I want all the layouts
```

After all, I do not want to know if you want to load them, or if you have a hardcoded list, or whatever...